spec/helper/revenuecat_internal_helper_spec.rb

```diff
@@ -593,4 +580,173 @@ def setup_stubs
       expect(tag_names).to include('5.10.0')
     end
   end
+
+  describe '.determine_next_version_using_labels' do
+    let(:repo_name) { 'purchases-ios' }
+    let(:server_url) { 'https://api.github.com' }
+    let(:http_method) { 'GET' }
+    let(:get_commits_response) do
```

Comment:
Hmm I'm not sure whether to extract them or not... If they did nothing (like `repo_name` or `server_url`), I would say to leave them inside since they don't have any logic and it doesn't hurt. These do have a bit of logic which would be good to avoid repetition of, but I also like having them close to the context of the tests... I'm thinking about creating a `read_test_file` method where we can just pass the name of the file and try to minify the repetition. What do you think? 

Also, the rename makes sense 👍